mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/MqttWebSocketClient.java

```diff
@@ -11,17 +10,15 @@
 import java.util.Optional;
 
 @Slf4j
-class GenericMqttWebSocketClient extends WebSocketClient {
+class MqttWebSocketClient extends WebSocketClient {
 
-    private final GenericMqttMessageConverter genericMqttMessageConverter = new GenericMqttMessageConverter();
+    private final MqttMessageListener listener;
 
-    private final GenericMqttMessageListener listener;
-
-    public GenericMqttWebSocketClient(URI serverUri) {
-        this(serverUri, new EmptyGenericMqttMessageListener());
+    public MqttWebSocketClient(URI serverUri) {
+        this(serverUri, new EmptyMqttMessageListener());
```

Comment:
suggestion: as EmptyMqttMessageLister is stateless and does nothing, should we enforce use of a singleton (define the static instance and private constructor in the nested class and use that)?

Reply:
No really related to my PR but changed.